autocomplete/definitions/namedTypes/mwseMCMSlider/convertToLabelValue/SkillSlider.lua

```diff
@@ -0,0 +1,14 @@
+local DistanceSlider = mwse.mcm.createSlider{
```

Comment:
A nit: can you change the variable name to `SkillSlider`, or don't store the result of `mwse.mcm.createSlider` call?